doc/sphinx/language/extensions/implicit-arguments.rst

```diff
@@ -491,12 +491,57 @@ in :ref:`canonicalstructures`; here only a simple example is given.
    This command supports the :attr:`local` attribute.  When used, the
    structure is canonical only within the :cmd:`Section` containing it.
 
-   Assume that :token:`qualid` denotes an object ``(Build_struct`` |c_1| … |c_n| ``)`` in the
-   structure :g:`struct` of which the fields are |x_1|, …, |x_n|.
-   Then, each time an equation of the form ``(``\ |x_i| ``_)`` |eq_beta_delta_iota_zeta| |c_i| has to be
+   Assume that :token:`qualid` denotes an object ``(Build_struct`` :n:`c__1` … :n:`c__n` ``)`` in the
```

Comment:
FTR, this can be made even simpler:

```suggestion
   Assume that :token:`qualid` denotes an object :n:`(Build_struct c__1 … c__n )` in the
```

Same for the rest below.

Reply:
Done, I think!

Reply:
Please check if this is resolved to your satisfaction.